tools/c7n_azure/tests/test_functionapp_utils.py

```diff
@@ -28,40 +29,39 @@ def setUp(self):
     @arm_template('functionapp-reqs.json')
     def test_get_storage_connection_string(self):
         storage_name = 'cloudcustodiantest'
-        conn_string = self.functionapp_util.get_storage_connection_string(
-            CONST_GROUP_NAME, storage_name)
-
+        id = '/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/resourceGroups/' \
```

Comment:
Just double checking it's ok to check in our sub id? We have code that replaces our sub id in the cassettes for a fake one. Not sure how you'd test this without the right one, but wanted to verify it wasn't a security risk and that's why we're replacing them in the cassettes.

Reply:
Actually now that I think about it I think it was so you could run this in other subs and the tests still pass.